web3.js/src/connection.ts

```diff
@@ -3229,14 +3428,25 @@ export class Connection {
 
   /**
    * Fetch a transaction details for a confirmed transaction
+   *
+   * @deprecated Deprecated since Solana v1.8.0. Please use {@link getTransaction} instead.
    */
   async getConfirmedTransaction(
     signature: TransactionSignature,
     commitment?: Finality,
   ): Promise<ConfirmedTransaction | null> {
-    const result = await this.getTransaction(signature, {commitment});
+    const args = this._buildArgsAtLeastConfirmed([signature], commitment);
+    const unsafeRes = await this._rpcRequest('getConfirmedTransaction', args);
```

Comment:
Should we update these `getConfirmed...` apis to use the newer RPC methods under the hood? Or too heavy-handed?

Reply:
I was worried about doing that right away.  I'm already worried about those calling `getBlock` / `getTransaction` since they'll start hitting a new endpoint, but clarity is more important here.

Maybe in a future release we do that?

Reply:
Wfm